src/main/java/seedu/mark/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.mark.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.mark.model.Model.PREDICATE_SHOW_ALL_BOOKMARKS;
+
+import seedu.mark.logic.commands.commandresult.CommandResult;
+import seedu.mark.logic.commands.exceptions.CommandException;
+import seedu.mark.model.Model;
+
+/**
+ * Reverts the {@code model}'s mark to its previously undone state.
```

Comment:
Let's change all the marks to Mark (consistent with the other code)
```suggestion
 * Reverts the {@code model}'s Mark to its previously undone state.
```